backend/src/backend/api/case_routes.py

```
@@ -0,0 +1,110 @@
+import json
+from datetime import datetime, timedelta
+from typing import List
```

Ruff (0.8.0)

`typing.List` imported but unused

Remove unused import: `typing.List`

(F401)